examples_tests/41.VisibilityBuffer/main.cpp

```diff
@@ -1191,6 +1192,26 @@ int main()
         }
     };
 
+    using DPG = ext::DepthPyramidGenerator::DepthPyramidGenerator;
+
+    DPG::Config config;
+    config.op = DPG::E_MIPMAP_GENERATION_OPERATOR::EMGO_BOTH;
+    config.outputFormat = DPG::E_IMAGE_FORMAT::EIF_R32G32_FLOAT;
+    DPG dpg(driver, am, depthBufferView, config);
+
+    const uint32_t mipCnt = DPG::getMaxMipCntFromImage(depthBufferView);
+    core::vector<core::smart_refctd_ptr<IGPUImageView>> mips(mipCnt);
+    DPG::createMipMapImageViews(driver, depthBufferView, mips.data(), config);
+
+    core::smart_refctd_ptr<IGPUDescriptorSetLayout> dpgDsLayout;
+    const uint32_t dpgDsCnt = DPG::createDescriptorSets(driver, depthBufferView, mips.data(), dpgDsLayout, nullptr, nullptr, config);
+    core::vector<core::smart_refctd_ptr<IGPUDescriptorSet>> dpgDs(dpgDsCnt);
+    core::vector<uint32_t> dpgPushConstants(dpgDsCnt);
```

Comment:
could you use an actual named `struct` with a `uint` member, instead of just a random `uint`